spec/support/request_helpers/endpoint_behaviour.rb

```diff
@@ -87,6 +87,38 @@ def accepts_root_path
         end
       end
     end
+
+    def validates_path_ownership
+      context "base_path has not already been registered" do
+        it "reserves the path for this publishing app" do
+          do_request
+
+          expect(PathReservation.count).to eq(1)
+          expect(PathReservation.first.base_path).to eq(base_path)
+          expect(PathReservation.first.publishing_app).to eq(content_item[:publishing_app])
+        end
+      end
+
+      context "base_path has already been registered" do
+        it "should be successful if the publishing app matches" do
+          expect{ do_request }.to change(PathReservation, :count).by(1)
```

Comment:
This should be testing against an existing PathReservation, and the count should not change.